data/src/main/java/org/cryptomator/data/cloud/crypto/CryptoImplDecorator.kt

```
@@ -332,16 +426,115 @@ abstract class CryptoImplDecorator(
 			}
+
+			closeQuietly(thumbnailReader)
 		} catch (e: IOException) {
 			throw FatalBackendException(e)
 		}
 	}
 
+	private fun closeQuietly(closeable: Closeable) {
+		try {
+			closeable.close();
+		} catch (e: IOException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)